style.css

```diff
@@ -0,0 +1,79 @@
+
+.title {
+	font-size: 10px;
+}
+
+body{ 
+	font-family:arial; 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/ad/pullRequest?prid=5498083)